lib/cmd_randip.go

```diff
@@ -0,0 +1,53 @@
+package lib
+
+import (
+	"math/rand"
+	"time"
+
+	"github.com/spf13/pflag"
+)
+
+// CmdRandIPFlags are flags expected by CmdRandIP.
+type CmdRandIPFlags struct {
+	Help bool
+	n    int
```

Comment:
Should be capital `N` otherwise if we wanted to access it from outside the `lib` package we wouldn't be able to.